pyrcareworld/pyrcareworld/envs/base_env.py

```diff
@@ -232,9 +232,9 @@ def close(self):
         """
         Close the environment
         """
-        if self.process is not None:
+        if hasattr(self, "process") and self.process is not None:
```

Comment:
I noticed during testing that there is a crash here if the environment closes before `__init__` finishes, which can happen if an exception is raised inside of `__init__`.